openg2p_registration/models/openg2p_registration.py

```diff
@@ -209,13 +247,15 @@ def _onchange_stage_id_internal(self, stage_id):
         return {'value': {'date_closed': False}}
 
     @api.model
+    @api.multi
     def create(self, vals):
         if vals.get('location_id') and not self._context.get('default_location_id'):
             self = self.with_context(default_location_id=vals.get('location_id'))
         if vals.get('user_id'):
             vals['date_open'] = fields.Datetime.now()
         if 'stage_id' in vals:
             vals.update(self._onchange_stage_id_internal(vals.get('stage_id'))['value'])
+        self.env['openg2p.beneficiary.school'].create(vals)
```

Comment:
Why are we saving duplicate data in "openg2p.beneficiary.school" model exactly? Also, this will always create an entry in the school model even if the current registration has nothing to do with school.